README.md

```diff
@@ -43,6 +32,3 @@ Give a ⭐️ if you like this project!
 - Inspiration
 - etc
 
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Kindly add a license file in your project and a license link in your readme which you have removed to make your readme more descriptive and to make your project licensed 🤔